frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -187,6 +219,7 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
         onEditClick={() => VMCDRomModal({ vmLikeEntity: vm, modalClassName: 'modal-lg' })}
         idValue={prefixedID(id, 'cdrom')}
         isNotAvail={cds.length === 0}
+        arePendingChanges={vmConfChanges['CD-ROMs']}
       >
         <DiskSummary disks={cds} vm={vm} />
```

Comment:
this will be buggy - the component expects VM cds and a VM. But is getting VMI cds and VM